dev-games/godot/godot-4.4.1-r1.ebuild

```diff
@@ -193,6 +194,8 @@ src_compile() {
 		target=$(usex tools editor template_$(usex debug{,} release))
 		dev_build=$(usex debug)
 
+		precision=$(usex double-precision double single)
```

Comment:
Please put this high up under `deprecated=$(usex deprecated)` without blank new lines.

Reminds me that I should merge this esconsargs+=() with the previous array to avoid confusion, this split was needed in the past but the code that it was done for was cleaned up from the ebuild.